ruby/bowling.rb

```diff
@@ -0,0 +1,48 @@
+class Bowling
```

Comment:
I expect that producing the ruby, then the JS version over the same weekend was a productive way to learn JS!